src/main/java/frc/trigon/robot/subsystems/shooter/ShooterCommands.java

```diff
@@ -0,0 +1,42 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.StartEndCommand;
+import frc.trigon.robot.RobotContainer;
+import org.trigon.commands.ExecuteEndCommand;
+import org.trigon.commands.NetworkTablesCommand;
+
+public class ShooterCommands {
+    public static Command getDebuggingCommand() {
+        return new NetworkTablesCommand(
+                (targetVelocity) -> ShooterCommands.getSetTargetVelocity(targetVelocity, targetVelocity),
+                false,
+                "Debugging/TargetDebuggingShootingVelocity"
+        );
```

Comment:
make this be able to get inputs individually for each side

Reply:
Wdym. Should it be two different commands?